logstash/defaults/main.yml

```diff
@@ -25,7 +25,7 @@ logstash_default_configs:
   - "30_pagerduty_output.conf"
 logstash_custom_configs: []
 
-logstash_remove_patterns: no
```

Comment:
for some stupid reasons I love the `yes`/`no` more :smile:

Reply:
It's kind of dangerous though. The canonical example is something like:

```country_codes:
  - us
  - uk
  - no
  - se
```

which breaks because yaml interprets the `no` there as a boolean rather than `norway`.